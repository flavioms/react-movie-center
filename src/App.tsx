import React from 'react';

// import { Container } from './styles';

const App: React.FC = () => {
  return (
    <div>
      <h2>Hello World</h2>
    </div>
  );
};

export default App;
