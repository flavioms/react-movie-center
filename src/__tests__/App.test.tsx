import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('Testing App Component', () => {
  it('render correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
