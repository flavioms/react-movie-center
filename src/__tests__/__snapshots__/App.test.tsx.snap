// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component render correctly 1`] = `
<div>
  <h2>
    Hello World
  </h2>
</div>
`;
